README.md

```diff
@@ -1,30 +1,43 @@
 # Cleanapp Backend version 2+
 
 This repository is for CleanApp (http://cleanapp.io) backend development.
-It's a complete rewrite after v.0.
 
-## Installation
+# Installation
```

Comment:
You are mixing dev workstation environment and test/prod envionment.

This was about #2. Separate, please, what is needed to develop into a separate section and, please, don't call it Installation.

Reply:
I think this comment is unrelated to the PR content. I think you commented changes related to the previous PR.